src/osyris/plot/map.py

```diff
@@ -338,10 +338,10 @@ def map(*layers,
         ndim=ndim)
 
     # Apply operation along depth
-    binned = getattr(binned, operation)(axis=1)
```

Comment:
I'm starting to think that operation shouldn't be a string but just a callable. Then you would pass whatever function you want, it could even be a custom function.

Reply:
How would the `map` function handle units in that case?

Reply:
I don't think we are handling the units well at the moment, so maybe we can come up with a better idea? See also #72

Reply:
Maybe we should just have a special top-level function for computing column densities, instead of relying on a special case of the thick map.

It would just be a thin wrapper around the `map` function, but this would make what happens to the units less ambiguous/hidden?

Reply:
I can give it a go in the coming weeks.
I'll close this PR and open a new one if that's okay with you.

Reply:
Alternatively, we can always merge this one for now and improve the mechanism later, since this is not breaking anything, just making you able to use `nansum` if you wish.

Reply:
Sure, let's do that in the meantime.